docs/resources/consul_snapshot.md

```diff
@@ -42,6 +42,7 @@ resource "hcp_consul_snapshot" "example" {
 - `restored_at` (String) Timestamp of when the snapshot was restored. If the snapshot has not been restored, this field will be blank.
 - `size` (Number) The size of the snapshot in bytes.
 - `snapshot_id` (String) The ID of the Consul snapshot
+- `snapshot_state` (String) The state of an HCP Consul snapshot.
```

Comment:
nit: `snapshot` in `snapshot_state` is redundant. Then again, there is `snapshot_id` so its consistent there.

If I were writing this from scratch I would name things as followed:

`id` -> `cluster_id`
`snapshot_id` -> `id`
`snapshot_state` -> `state`

For now though, I think I would prefer naming this `state` instead but feel free to ignore this comment and continue with the current name if you have strong feelings otherwise.

Reply:
using just `state` matches our model: https://github.com/hashicorp/cloud-consul-service/blob/de4939ba1d7f47c9f002ec74dc34228fe992fa7e/proto-public/hashicorp/cloud/consul_20210204/consul.proto#L157